### ISSUE TITLE
Suggestion: more prominent highlighting for selected command palette item

### ISSUE THREAD

**Issue by rkpatel33** [Monokai/monokai-pro-vscode#73]
In VSCode, I find it difficult to discern the selected item in the command palette (and the other menus that use the same selection component). I think it would be helpful if it had a lighter background color as it does in the Sublime Text version, or a more prominent font style/color/weight.

FYI, I bought the ST3 and VSC versions and love them. Thx.

![image](https://user-images.githubusercontent.com/3799489/40150838-e9d2c03c-5930-11e8-94d1-466925f56757.png)

![image](https://user-images.githubusercontent.com/3799489/40150862-153b7bd8-5931-11e8-9a97-014934aec8bc.png)

**Comment by rkpatel33** [comment]
FYI have solved this for the time being by setting some theme values as follows:
```
"workbench.colorCustomizations": {
        "list.focusBackground": "#383E4A", // command palette highlighted item
        "list.hoverBackground": "#292d35", // command palette mouse hover highlight
}
```

**Comment by Monokai** [comment]
Fixed in 1.1.7
